public/javascripts/services.js

```diff
@@ -259,15 +269,38 @@ services.factory('CommodityFactory', function($resource) {
     })
 });
 
+services.factory('CommodityEditFactory', function($resource){
+    return $resource('farmer/:id/commodity/:commodity_id', {}, {
+        update: { method: 'PUT', params: {id: '@id', commodity_id: '@commodity_id'} },
```

Comment:
@nick22891 remove the comma from this line... how are your factories able to function with this comma?